ext/fetch/26_fetch.js

```diff
@@ -200,6 +200,8 @@
     }
     terminator[abortSignal.add](onAbort);
 
+    let requestSendError;
+    let requestSendErrorSet = false;
```

Comment:
So cumbersome to use 2 variables for this.
If you're concerned that someone might throw undefined you could use a magic value (e.g. A symbol) to indicate "no error".